index.js

```diff
@@ -0,0 +1,68 @@
+const readline = require('readline-sync');
+const parse = require('csv-parse/lib/sync');
+const log4js = require('log4js');
+const logger = log4js.getLogger('logs\\debug.log');
+const fs = require('fs');
+const splitPoundsPennies = require('./splitPoundsPennies');
+const createAccountArray = require('./createAccountArray');
+const addSubtractInAccounts = require('./addSubtractInAccounts');
+const parseIntTransacAmounts = require('./parseIntTransacAmounts');
+const recombinePenniesPounds = require('./recombinePenniesPounds');
+const userRequestList = require('./userRequestList');
+
+log4js.configure({
+    appenders: {
+        file: { type: 'fileSync', filename: 'logs\\debug.log' }
+    },
+    categories: {
+        default: { appenders: ['file'], level: 'debug'}
+    }
+});
+
+class Transaction {
+    constructor(Date,FromAccount,ToAccount,Narrative,Amount,amountPounds,amountPennies){

```

Comment:
It'd be better to use lowercase names (`date, fromAccount, toAccount, ...`), both for consistency with JS in general, and in particular here because `Date` is the global Date class, which is hidden inside your constructor by the parameter of the same name!